packages/storybook/src/button.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ArgsTable, Primary, PRIMARY_STORY, Stories } from '@storybook/addon-docs';
+import { Meta, StoryObj } from '@storybook/react';
+import { Button } from '../../components-react/src/Button';
+
+const meta = {
+  title: 'React Component/Button',
+  id: 'react-button',
+  component: Button,
+  args: {
+    children: 'Read more...',
+  },
+  parameters: {
+    tokensPrefix: 'utrecht-button',
```

Comment:
At first glance I'd say the prefix should be `example-button`, not `utrecht-button`.

However, the `tokensPrefix` is only meaningful when we actually display the tokens, and in Utrecht this is done with a custom component. Since we don't yet support automatic design token docs in the example repo, I think we shouldn't include it here.

Reply:
Removed this.